server/webhook.go

```diff
@@ -14,6 +14,7 @@ import (
 	"path"
```

Comment:
Is it possible to split the change into several smaller pieces? All changes in github/gitlab and vcs.go are all good. I see several refactoring changes in webhook.go and can we split the change into two changes? One for refactoring and the other one for actual change for sql review. Thanks.